### PR TITLE
feat: add German language support

### LINKED ISSUE #240: Add German language support
I can provide a PR as native speaker.

### PR DESCRIPTION
**Pull request by jhoermann**
Closes #240

### CODE REVIEW

**Review by fmaclen — CHANGES_REQUESTED** [review]
Thanks again for putting this together.

Now that #234 has been merged there were significant changes to the locales, a handful of keys were removed and several more were added.

Here's the specific list of changes:
https://github.com/fmaclen/hollama/pull/234/files#diff-dfa2062fc0f5cfa7c4ef6179d7480621e7f0ab7a8a0e51fe25df994871186146

File: src/routes/settings/Interface.svelte [hunk 1]
```diff
@@ -29,11 +29,12 @@
 		onChange={() => changeLanguage(value)}
 		options={[
 			{ value: 'en', label: 'English' },
+			{ value: 'zh-cn', label: '中文 (简体)' },
 			{ value: 'es', label: 'Español' },
+			{ value: 'pt-br', label: 'Português (Brasil)' },
+			{ value: 'de', label: 'Deutsch' },
```
[the remark this answers is not in the document]
Comment (jhoermann) [reply src/routes/settings/Interface.svelte]: Changed now.

File: tests/settings.test.ts [hunk 2]
```diff
@@ -240,4 +240,20 @@ test.describe('Locales', () => {
...
+
+			await page.evaluate(() => window.localStorage.clear());
+			await page.reload();
+			await expect(page.getByText('Current version')).not.toBeVisible();
+			await expect(page.getByText('Aktuelle Version')).toBeVisible();
+			expect(await page.evaluate(() => window.localStorage.getItem('hollama-settings'))).toContain(
+				'"userLanguage":"de"'
+			);
+		});
+	});
```
[the remark this answers is not in the document]
Comment (jhoermann) [reply tests/settings.test.ts]: Moved now - still need to adapt the messages though.

**Comment by jhoermann** [comment]
Did all the changes - can you please check again @fmaclen?

**Review by fmaclen — APPROVED** [review]
This looks great, thanks @jhoermann!

**Pull request merged**